src/pages/home/base/mobile-core/identity/tabs/index.md

```diff
@@ -2,7 +2,33 @@
 noIndex: true
 ---
 
-<Variant platform="android" task="add" repeat="3"/>
+<Variant platform="android" task="add" repeat="10"/>
+
+1. Add the `Identity` library to your project using the app's gradle file.
+
+#### Kotlin
+
+```kotlin
+implementation(platform("com.adobe.marketing.mobile:sdk-bom:3.+"))
+implementation("com.adobe.marketing.mobile:core")
+implementation("com.adobe.marketing.mobile:identity")
+```
+
+#### Groovy
+
+```java
+implementation platform('com.adobe.marketing.mobile:sdk-bom:3.+')
+implementation 'com.adobe.marketing.mobile:core'
+implementation 'com.adobe.marketing.mobile:identity'
+```
+
+<InlineNestedAlert variant="warning" header="false" iconPosition="left">
+
+Using dynamic dependency versions is **not** recommended for production apps. Please read the [managing Gradle dependencies guide](../../resources/manage-gradle-dependencies.md) for more information.
+
+</InlineNestedAlert>
```

Comment:
can we use the shared alert - same for all the updated files
```
import Alerts from '/src/pages/resources/alerts.md'
...
<Alerts query="platform=android-gradle&componentClass=InlineNestedAlert"/>
```

Reply:
I did not know we can import variants from the tabs page.

Reply:
The Alert uses relative path to link to the managing-gradle-dependencies page and it does not work well for Core extensions. 

Let me know if there is a way to use absolute paths.